enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationRestService.java

```diff
@@ -0,0 +1,653 @@
+    public ObjectResponse<DocumentStatusResponse> checkDocumentStatus(ObjectRequest<DocumentStatusRequest> request,
+                                                                      EciesEncryptionContext eciesContext,
+                                                                      PowerAuthApiAuthentication apiAuthentication)
+            throws PowerAuthAuthenticationException, PowerAuthEncryptionException, OnboardingProcessException {
+
+        final String operationDescription = "checking document verification status";
+        checkApiAuthentication(apiAuthentication, operationDescription);
+        checkEciesContext(eciesContext, operationDescription);
+        checkRequestObject(request, operationDescription);
+
+        final OwnerId ownerId = PowerAuthUtil.getOwnerId(apiAuthentication);
+        final String processId = request.getRequestObject().getProcessId();
+
+        logger.debug("Onboarding process will not be locked, {}", processId);
+        onboardingService.verifyProcessId(ownerId, processId, OnboardingStatus.VERIFICATION_IN_PROGRESS);
+
+        final DocumentStatusResponse response = identityVerificationService.fetchDocumentStatusResponse(request.getRequestObject(), ownerId);
+        return new ObjectResponse<>(response);
+    }
+
+    /**
+     * Initialize document verification SDK for an integration.
+     * @param request Presence check initialization request.
+     * @param eciesContext ECIES context.
+     * @param apiAuthentication PowerAuth authentication.
+     * @return Verification SDK initialization response.
+     * @throws PowerAuthAuthenticationException Thrown when request authentication fails.
+     * @throws PowerAuthEncryptionException Thrown when request decryption fails.
+     * @throws DocumentVerificationException Thrown when SKD initialization fails.
+     * @throws OnboardingProcessException Thrown when onboarding process identifier is invalid.
+     * @throws RemoteCommunicationException In case of remote communication error.
+     */
+    public ObjectResponse<DocumentVerificationSdkInitResponse> initVerificationSdk(
+            ObjectRequest<DocumentVerificationSdkInitRequest> request,
+            EciesEncryptionContext eciesContext,
+            PowerAuthApiAuthentication apiAuthentication)
+            throws PowerAuthAuthenticationException, DocumentVerificationException, PowerAuthEncryptionException, OnboardingProcessException, RemoteCommunicationException {
+
+        final String operationDescription = "initializing document verification SDK";
+        checkApiAuthentication(apiAuthentication, operationDescription);
+        checkEciesContext(eciesContext, operationDescription);
+        checkRequestObject(request, operationDescription);
+
+        final OwnerId ownerId = PowerAuthUtil.getOwnerId(apiAuthentication);
+        final String processId = request.getRequestObject().getProcessId();
+
+        logger.debug("Onboarding process will not be locked, {}", processId);
```

Comment:
I know, it's debug only, but probably highlight to distinguish from `Onboarding process will be locked`
```suggestion
        logger.debug("Onboarding process will NOT be locked, {}", processId);
```

Reply:
I don't like all-caps in log messages. In case we need to use these debug messages I am sure we will be able to differentiate the locked and not-locked cases.